actions/views/channel.ts

```diff
@@ -179,7 +181,7 @@ export function leaveChannel(channelId: string) {
             dispatch(selectTeam(''));
             dispatch({type: TeamTypes.LEAVE_TEAM, data: currentTeam});
             browserHistory.push('/');
-        } else {
+        } else if (channelId === currentChannelId) {
```

Comment:
Since we're changing non-insights code, I think a comment about why we're doing this might help other teams working on this.